covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -36,31 +42,53 @@ def run_server(data_dir: Path, simulated_polynomial: Path, simulated_exponential
     """Creates and runs a flask server."""
     server = Flask(__name__, template_folder=CURRENT_DIR)
 
-    covid19_normal_app = CountryGraph.create_dashboard(data_dir, server, GraphType.Normal)
-    covid19_semilog_app = CountryGraph.create_dashboard(data_dir, server, GraphType.SemiLog)
-    covid19_loglog_app = CountryGraph.create_dashboard(data_dir, server, GraphType.LogLog)
-    covid19_heatmap_app = HeatMap(str(simulated_polynomial)).create_app(server)
-    covid19_heatmap_exponential_app = HeatMap(str(simulated_exponential)).create_app(server)
+    _create_prediction_apps(server=server, data_dir=data_dir)
+    _create_simulation_apps(server, simulated_polynomial, simulated_exponential)
 
     @server.route("/")
     def home():
         return render_template("index.html")
 
     @server.route("/covid19/")
```

Comment:
This is the URL that we pasted all over the internet, we should redirect that one.

```suggestion
    @server.route("/covid19/normal")
```

Reply:
Yep, good idea. Slightly unrelated, why do we route everything through `/covid19/` prefix?

Reply:
The prefix is there, since the full URL is `https://graphs.<domain>/covid19`. If we take it away, COVID-19 would not be present in the URL, which I think is wrong. I might use the subdomain for another purpose at some point. You might argue that the prefix should be added by Nginx instead - I'd agree with it, but that's extra work. Or we can change the subdomain to `covid-graphs.<domain>`, but I think we've missed that train :smile:

This is yet another episode of Naming is Hard.